content/extraProblems.json

```diff
@@ -472,6 +472,110 @@
         "usacoId": "1142"
       }
     },
+    {
+      "uniqueId": "usaco-1156",
+      "name": "Air Cownditioning",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1156",
+      "source": "Bronze",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Greedy"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1156"
+      }
+    },
+    {
+      "uniqueId": "usaco-1159",
```

Comment:
If you don't want to add this to DFS module, maybe add it to the conclusion problems module

Reply:
Can't recall having ever added anything to the conclusion problems, you can do in separate PR if you want.